extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/internal/WemoHandlerFactory.java

```diff
@@ -45,6 +48,7 @@
  * @author Hans-Jörg Merk - Initial contribution
  * @author Kai Kreuzer - some refactoring for performance and simplification
  */
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.wemo")
```

Comment:
Do you think something will break because of this change? I added the PID so it follows the convention that is also used with the bindings generated with the skeleton.

[__bindingIdCamelCase__HandlerFactory.java](https://github.com/eclipse/smarthome/blob/95830210a7f215823712f0fbca053e938e878950/tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__HandlerFactory.java#L43):
```java
@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.${bindingId}")
```